src/Button.vue

```diff
@@ -115,12 +118,14 @@ export default {
     },
 
     distance () {
-      return px(this.width - this.height + constants.margin)
+      const buttonLabelWidth = this.labels.button ? this.labels.button.length * 3 : 0;
+      return px(this.width + constants.margin - ( this.height + buttonLabelWidth) )
     },
 
     buttonStyle () {
+      const buttonLabelWidth = this.labels.button ? this.labels.button.length * 3 : 0;
```

Comment:
ditto

Reply:
Yes (magic), but not sure how to calc the word length in pixels.
String pixel length varies on the same length.
Example - both 4 latter words:
guss
join

This is also why the button text supports up to 10 chars.

Needs a solution to calculate word length in pixels

Reply:
Hi @euvl,
fixed button distance transformX width by getting the actual div width.
so now even bigger and longer text should be supported.

only downside is using the query selector. but in general the performance should suffer much because of several query selectors